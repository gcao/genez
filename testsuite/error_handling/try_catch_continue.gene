# Test execution continues after catch
(println "Before try")

(try
  (do
    (println "In try")
    (throw "Error!")
    (println "Should not print"))
  (catch e
    (println "Caught:" e)))

(println "After try/catch")