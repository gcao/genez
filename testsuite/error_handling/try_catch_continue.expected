Before try
In try
Caught: Error!
After try/catch